ideascube/models.py

```diff
@@ -311,6 +311,13 @@ def get_prep_value(self, value):
         return json.dumps(value)
 
 
+class LanguageField(models.CharField):
+    def __init__(self, *args, **kwargs):
+        kwargs['choices'] = get_all_languages()
+
```

Comment:
>> Yet the former is much less readable.
>
> That is subjective, and it's my purpose here. ;)

I disagree. Elegance is subjective, readability isn't. But even if it were, does it actually matter?

You're not going to convince me that compact, blank-lines-free code is more readable, and maybe I won't convince you of the opposite. Does it actually matter?

We are individuals, we have our own subjectivity, and yes, sometimes that will show in the code. But let's not make the opposite mistake of pushing towards uniformity.

To quote pep8 itself:

> A Foolish Consistency is the Hobgoblin of Little Minds

---

> So I think we should follow a mantra

I think mantras are generally a bad idea, as they inevitably push us towards inflexibility and following them as if they were hard rules. Instead we should just think for ourselves and write clear code, often on a case by case basis. Sometimes it will mean adding white lines where they aren't dictated by pep8, and sometimes it won't matter at all.

I started this thread not caring, but through what I feel is a completely useless squabble over trivialities, you're honestly pushing me towards actually caring and wanting to stick to it.

Some things do matter, but this isn't one of them.

Feel free to remove that line if it bothers you that much. I feel like I've already wasted way too much time over this, so this will be my last reply in this thread.